src/Components/App/App.js

```diff
@@ -22,13 +21,25 @@ function App() {
         }
       })
     }, [])
+    
+  const updateSelectedMovie = (event) => {
+    getMovie(event.target.id || event.target.parentNode.id || event.target.parentNode.parentNode.id)
+    .then(data => {
+      if (!data) {
+        setServerError(true)
```

Comment:
Another option here is to utilize the error message returned from the server. Instead of relying on a simple true/false value, you can set the initial error as an empty string and update it within your fetch to reflect the error message received from the server. It's important to note that not all errors from the server are related to server downtime. Sometimes, users may have sent incorrect data to the server, resulting in 400 type errors. In such cases, we want to provide users with feedback to help them correct their input.

Reply:
Oh true, that's very helpful